x-pack/plugins/rule_registry/common/assets/field_maps/technical_rule_field_map.ts

```diff
@@ -16,6 +16,7 @@ export const technicalRuleFieldMap = {
     Fields.EVENT_ACTION,
     Fields.TAGS
   ),
+  [Fields.ALERT_RULE_PARAMS]: { type: 'keyword', index: false },
```

Comment:
Curious: How do we decide whether a field should be required or not?

Reply:
In this case I decided not to index it because it's a serialized field, so I don't think it made sense to allow users to search on it.